.github/workflows/pack.yml

```diff
@@ -1,9 +1,6 @@
 on:
   push:
-    branches:
-    - main
-    - development
-    - rc-*
+    branches: ['*']
```

Comment:
What is the reason for targeting all branches?

Reply:
this pr is testing for gh actions. i'll close this.